README.md

```diff
@@ -7,18 +7,54 @@
 
 **[Read it here]**.
 
-This _Rust Cookbook_ is a collection of simple examples that
+This _Rust Cookbook_ is a collection of simple [Rust] examples that
 demonstrate good practices to accomplish common programming tasks,
 using the crates of the Rust ecosystem.
 
 These examples are complete, and suitable for copying directly into
 new cargo projects. They are tested and guaranteed to work.
 
+## Read it offline
+
+If you'd like to read it locally:
+
+```bash
+$ git clone https://github.com/rust-lang-nursery/rust-cookbook
+$ cd rust-cookbook
+$ cargo install mdbook --vers "0.0.28"
+$ mdbook build
+```
+
+The output will be in the `book` subdirectory. Open it in your web browser.
```

Comment:
\<bikeshedding>
For this part it's probably going to be easier to say "open it with the default web browser", then the next three snippets go from iterating through all the possible browsers to using a OS-specific program for opening files with the default application.

```
$ xdg-open ./book/index.html # linux
$ start .\book\index.html    # windows
$ open ./book/index.html     # mac
```

It may be simpler to just use the `--open` flag when building the book.

```
$ mdbook build --open
```

\</bikeshedding>

Reply:
Nice and simple- excellent